test/integration/annotation_spec.mjs

```diff
@@ -503,6 +503,12 @@ describe("ResetForm action", () => {
       it("must check that the Ink annotation has a popup", async () => {
         await Promise.all(
           pages.map(async ([browserName, page]) => {
+            // TODO: re-enable this test when the Ink annotation has been made
+            // editable.
+            if (browserName) {
+              return;
+            }
```

Comment:
This can easily be forgotten about with just a TODO comment, hence we should handle this as usual for disabled tests.
```suggestion
            if (browserName) {
              pending("Re-enable this test when the Ink annotation has been made editable.");
            }
```